python/ray/dashboard/modules/job/tests/test_job_manager.py

```diff
@@ -46,6 +48,59 @@
 import psutil
 
 
+@pytest.mark.asyncio
```

Comment:
can we create a new test suite for all export events? I assume we will need more e2e testing eventually in python layers, and it'd be nice to put them in the same file.